<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorSelector</class>
 <widget class="QMainWindow" name="colorSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>334</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>669</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="selectedColour">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="OpacitySlider">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>22</width>
      <height>249</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="OpacityExample">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>44</width>
      <height>249</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
background: qlineargradient( x1:0 y1:0, x1:0 y1:1, stop:0 rgba(0,0,0,0), stop:1 rgba(0,0,0,255) )
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="RedExample">
       <property name="styleSheet">
        <string notr="true">* {
background: qlineargradient( x1:0 y1:0, x1:0 y1:1, stop:0 rgba(0,0,0,255), stop:1 rgba(255,0,0,255) )
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="RedSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="GreenExample">
       <property name="styleSheet">
        <string notr="true">* {
background: qlineargradient( x1:0 y1:0, x1:0 y1:1, stop:0 rgba(0,0,0,255), stop:1 rgba(0,255,0,255) )
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="GreenSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="BlueExample">
       <property name="styleSheet">
        <string notr="true">* {
background: qlineargradient( x1:0 y1:0, x1:0 y1:1, stop:0 rgba(0,0,0,255), stop:1 rgba(0,0,255,255) )
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSlider" name="BlueSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
